payments/billing_sdk/src/main/java/com/aptoide/pt/billing_sdk/AppcoinsBillingService.kt

```diff
@@ -0,0 +1,12 @@
+package com.aptoide.pt.billing_sdk
```

Comment:
This doesn't look right, since this module is meant to be shared with the Wallet.
Mb use the same package as the aidl?